msteams-ui-styles-core/src/colors.ts

```diff
@@ -37,6 +42,7 @@ export const Colors = {
     orange04: '#CC4A31', // new
     yellow: '#F8D22A',
     green: '#92C353',
+    green1: '#237B4B',
```

Comment:
What is green1? Send me the redlines that reference it

Reply:
green1 colour is using for success icon(successIcon: colors.light.green1) in inputs field

Reply:
green1 colour is not mentioned in the colour ramp. I implemented colour like this ('green1: '#237B4B', // don't know'.) in the ts file.

Reply:
just replace the value of existing green

Reply:
Replaced existing green instead of the new one